src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -1684,9 +1685,10 @@ public void doTimeout(@QueryParameter String token) {
     public static final class CustomJenkinsRule extends JenkinsRule implements AutoCloseable {
         private final URL url;
 
-        public CustomJenkinsRule(URL url) throws Exception {
+        public CustomJenkinsRule(URL url, String contextPath) throws Exception {
```

Comment:
Since the class and constructor are public, I think you should keep the old constructor signature and mark it deprecated.